README.md

```diff
@@ -44,6 +44,7 @@ start by looking at [ADOPTERS.md](ADOPTERS.md).
 
 - [Github discussion forum](https://github.com/containers/bootc/discussions) for async discussion
 - [#bootc-dev on CNCF Slack](https://cloud-native.slack.com/archives/C08SKSQKG1L) for live chat
+- Recurring [Jitsi live meeting](https://meet.jit.si/YourOperatingSystemIsAContainer) each Friday at 11:30 EDT/ 15:30 UTC.  (Currently ambiguous what happens during daylight saving time switch, if in doubt ask in Slack!)
```

Comment:
Unclear if we want to bikeshed what happens when dst switches here.  Also this currently references summer time.

Timezones are hard.

Reply:
my votes are 

- use UTC
- Use CNCF zoom

Reply:
And skip recording for now but we keep a summary of things and then be sure we've updated any issues/PRs that were discussed

Reply:
Do we still need to sort out getting access to the CNCF zoom?  I see https://github.com/cncf/sandbox/issues/339#issuecomment-2730445555 but nothing else in that thread about it.  Not sure if it was handled elsewhere and I'm just missing it.

Reply:
They gave me access and I can give access to as many hosts as needed it seems.

Reply:
Ok I've updated this to use the Zoom link and to refer to the time only as UTC.